### ISSUE TITLE
Modernize the look and feel of NepalMap

### ISSUE THREAD

**Issue by cliftonmcintosh** [Code4Nepal/nepalmap_app#243]
Counting India has very nice modern layout that looks much like Google's Material Design approach. See https://www.countingindia.com/profiles/country-IN-india/. Notice how each section has a card-like quality that is more modern than the older style that we are using.

Determine if we can update NepalMap with similar improvements. If not, explain why. If we can, submit a design proposal before proceeding with an update so that we can verify if the team wants to use the updated design.

**Comment by ravinepal** [comment]
Thank you @cliftonmcintosh. I support the idea of thinking about new designs for NepalMap. @arlexgole would you like to advise or support us on this? Thank you

**Comment by devbks** [comment]
Agreed @cliftonmcintosh 
NepalMap ui looks old, i'll work on some prototype

**Comment by arlexgole** [comment]
@ravinepal I can help out. Is the focus just on updating the UI?

**Comment by ravinepal** [comment]
Thank you

> @ravinepal I can help out. Is the focus just on updating the UI?

Thank you, @arlexgole. It'd be great to update UI & IX. 

I'd prefer to keep the same color schemes. These are our brand color schemes.

Red: #E00015
Blue: #00ADEF
White: #FFFFFF light gray: #FCFBFB

**Comment by arlexgole** [comment]
Let me know if you have any feedback and I can update the designs. 

![home](https://user-images.githubusercontent.com/32579978/52686266-ed653980-2f1a-11e9-9651-5718671a932e.png)

![dashboard](https://user-images.githubusercontent.com/32579978/52686268-efc79380-2f1a-11e9-8a45-3134f556555f.png)

**Comment by ravinepal** [comment]
Thank you, @arlexgole! much appreciated. I like it. @cliftonmcintosh @pratimakandel @devbks what do you guys think?

**Comment by cliftonmcintosh** [comment]
Yes, those are nice improvements. Thank you very much, @arlexgole, for working on this. Is this something that can be updated directly in our projects without forking/hacking the underlying wazimap framework?

**Comment by cliftonmcintosh** [comment]
@arlexgole 

Are you still interested in taking this on?

**Comment by arlexgole** [comment]
Yes but I am not too sure about how the wazimap framework works. Is there an engineer who I can work with to try to possibly implement the designs? I can also try to look into the code myself.

**Comment by ishan-dongol** [comment]
I can help you with that @arlexgole . @ravinepal
